### PR TITLE
Re adding a feature flag used by the front end

### PR DESCRIPTION
**Pull request by darkbushido**
## Description of change
Re-Adding the feature flag for the front end

## Original issue(s)
department-of-veterans-affairs/va.gov-team#14154

## Things to know about this PR
* Feature Flag `facilities_ppms_suppress_pharmacies`

### CODE REVIEW

File: config/features.yml [hunk 1]
```diff
@@ -38,12 +38,15 @@ features:
   facility_locator_ppms_legacy_urgent_care_to_pos_locator:
     actor_type: user
     description: force the legacy urgent care path to use the new POS locator
-  facility_locator_ppms_location_query:
+  facility_locator_ppms_use_v1_client:
```
Comment (thilton-oddball) [inline config/features.yml:41]: Just double checking that you also wanted to change 'facility_locator_ppms_location_query' to 'facility_locator_ppms_use_v1_client'

**Review by thilton-oddball — APPROVED** [review]
One comment that should be reviewed but otherwise looks good.

File: config/features.yml (hunk 1 above)
Comment (darkbushido) [reply config/features.yml:41]: yep, `: facility_locator_ppms_location_query` was removed with its logic and replaced with `: facility_locator_ppms_use_v1_client`

Comment (darkbushido) [reply config/features.yml]: actually, this change belongs in a different branch. fixing it

**Pull request merged**